website/docs/externaldata.md

```diff
@@ -458,7 +458,7 @@ To ensure a request to the Gatekeeper webhook is coming from the API server, Gat
 
 1. Deploy Gatekeeper with a client CA cert name. Provide name of the client CA with the flag `--client-cert-name`. The same name will be used to read certificate from the webhook secret. The webhook will only authenticate API server requests if client CA name is provided with flag.
 
-2. You will need to patch the webhook secret manually to attach client ca crt. Update secret `gatekeeper-webhook-server-cert` to include `clientca.crt`. Key name `clientca.crt` should match the name passed with `--client-cert-name` flag.
+2. You will need to patch the webhook secret manually to attach client ca crt. Update secret `gatekeeper-webhook-server-cert` to include `clientca.crt`. Key name `clientca.crt` should match the name passed with `--client-cert-name` flag. For client CA you could either use default kubernetes general ca located at `\etc\kubernetes\pki\ca.crt`, or you could generate your own CA for this purpose.
```

Comment:
```suggestion
2. You will need to patch the webhook secret manually to attach client ca crt. Update secret `gatekeeper-webhook-server-cert` to include `clientca.crt`. Key name `clientca.crt` should match the name passed with `--client-cert-name` flag. For client CA you could either use the default Kubernetes CA located at `\etc\kubernetes\pki\ca.crt`, or you could generate your own CA for this purpose.
```

Reply:
We need to stop recommending that people mix trust domains.

Reply:
Got it!